cmake/DAQ.cmake

```diff
@@ -333,6 +333,81 @@ function(daq_codegen)
   set(DAQ_PROJECT_GENERATES_CODE true PARENT_SCOPE)
 endfunction()
 
+####################################################################################################
+# daq_protobuf:
```

Comment:
I suspect it would be helpful to have one of "gen", "codegen" in the function name